src/main/java/javaslang/collection/Iterator.java

```diff
@@ -69,7 +70,6 @@ public Object next() {
      * @param <T>       Component type.
      * @return A new {@code javaslang.collection.Iterator}
      */
-    @SuppressWarnings("varargs")
```

Comment:
It is good to remove unnecessary stuff. After the pull I will test it again in Eclipse, IntelliJ etc. I had cases where one IDE says it is ok, the other complains... We will see :-)